docs/man_pages/project/configuration/resources-update.md

```diff
@@ -0,0 +1,27 @@
+resources-update
```

Comment:
That's right. You need to put this header, in order the article to be part of docs.nativescript.org (otherwise, it will be just skipped):

```
<% if (isJekyll) { %>---
title: <some title>
position: <order number>
---<% } %>
```
You can take a look at any of the other articles for an example. 

As a side note, put a # sign before the title to make it look ok in html.